docs/en/advanced-topics/flashing-from-python-code.rst

```diff
@@ -0,0 +1,52 @@
+Flashing from your python code
+==============================
+
+The following is an example on how to flash the ESP32 from a custom application:
```

Comment:
Changing `ESP32` to `{IDF_TARGET_NAME}` will make it dynamically change based on the selected target.